source/about/cloud-subscriptions.rst

```diff
@@ -4,27 +4,83 @@ Cloud subscriptions
 Purchase a Cloud subscription
 ------------------------------
 
-Mattermost Cloud subscriptions start with a 14-day trial. When you're near the end of your trial, a banner within Mattermost will notify you to upgrade to a subscription plan.
+Mattermost Cloud can be used free forever with Mattermost Starter.
+
+You can also start a free, 30-day, trial of Mattermost Enterprise to explore additional features. When you're near the end of your trial, a banner within Mattermost will notify you to upgrade to a subscription plan. If you'd like to continue with Mattermost Enterprise, please contact our `Sales team <mailto:sales@mattermost.com>`_. 
+
+If, after 30 days, you choose not to continue using Mattermost Enterprise, your subscription will revert to Mattermost Starter. 
+
+Alternatively, you can upgrade to a monthly per-user subscription, with Mattermost Professional.
```

Comment:
Propose we make upgrading to Professional the primary action:

----

When you're near the end of your trial, a banner within Mattermost will notify you to upgrade to a subscription plan:
1. Select Subscribe now from the banner.
2. Provide your credit card details.
  - Your credit card will be verified and you'll be charged at the end of the billing period. If you'd prefer an annual Cloud subscription, please contact our Sales team for a quote.
3. Provide your billing address.
4. Select the Professional subscription plan.
5. Select Subscribe.

Alternatively, if you'd like to continue with Mattermost Enterprise, please contact our [Sales team](mailto:sales@mattermost.com).

If you choose not to upgrade at the end of your 30-day trial, your subscription will revert to Mattermost Starter.